src/show.jl

```diff
@@ -71,6 +71,95 @@ function Base.string(
     return str
 end
 
+"""
+    string(x::ArbOrRef; digits, more, no_radius, condense, unicode, remove_trailing_zeros)
+
+Convert `x` to a decimal string. By default, this uses the
+midpoint-radius format "[m ± r]".
+
+With default flags, the output can be parsed back as `Arb(string(x))`,
+and this is guaranteed to produce an interval containing the original
+interval `x` (but is generally wider). For loss free, but not human
+readable, serialization as a string, see [`dump_string`](@ref) and
+[`load_string`](@ref).
+
+By default, the output is rounded so that the value given for the
+midpoint is correct up to 1 ulp (unit in the last decimal place).
+
+!!! warning
+    The output can be confusing for wide inputs, when the radius is too
+    large for even the first digit to be known up to 1 ulp. In this case
+    the interval is printed in the format `[+/- R]` where `R` is an upper
+    bound for the absolute value of `x`. The `more` argument (see below)
+    can then be used (possibly in combination with `digits`) to print more
+    digits, but which are then not guaranteed to be correct within 1 ulp.
+    ```jldoctest
+    julia> x = Arb((1, 2));
+
+    julia> string(x)
+    "[+/- 2.01]"
+
+    julia> string(x, more = true)
+    "[1.5000000000000000000000000000000000000000000000000000000000000000000000000000 +/- 0.501]"
+
+    julia> string(x, digits = 5, more = true)
+    "[1.5000 +/- 0.501]"
+    ```
+    The [`getball`](@ref) and [`getinterval`](@ref) functions can also
+    be useful in this case.
+
+# Keyword Arguments
+- `digits::Integer = digits_prec(precision(x))`: The number of digits
+  to display.
+- `more::Bool = false`: If `true`, display more (possibly incorrect)
+  digits.
+- `no_radius::Bool = false`: If `true`, display the radius is not
+  included in the output. Unless `more` is set, the output is rounded
+  so that the midpoint is correct to 1 ulp. As a special case, if
+  there are no significant digits after rounding, the result will be
+  shown as ``0e+n``, meaning that the result is between ``-1e+n`` and
+  ``1e+n`` (following the contract that the output is correct to
+  within one unit in the only shown digit).
```

Comment:
```suggestion
- `no_radius::Bool = false`: If `true`, the radius is not
  displayed in the output. Unless `more` is set, the output is rounded
  so that the midpoint is correct to 1 ulp. As a special case, if
  there are no significant digits after rounding, the result will be
  shown as ``0e+n``, meaning that the result is between ``-1e+n`` and
  ``1e+n`` (following the contract that the output is correct to
  within one unit in the only shown digit).
```